docs/api/commands/intercept.mdx

```diff
@@ -607,6 +611,17 @@ cy.intercept('POST', 'http://api.company.com/widgets', (req) => {
 // real destination
 ```
 
+### Disabling logs for a request
```

Comment:
Is this something worth mentioning in a guide as well? It seems like a very useful pattern, and I wouldn't know to look in the documentation to `cy.intercept()` to find it. Maybe in the FAQ?